fundingmanager_test.go

```diff
@@ -687,16 +711,38 @@ func waitForOpenUpdate(t *testing.T, updateChan chan *lnrpc.OpenStatusUpdate) {
 
 func assertNoChannelState(t *testing.T, alice, bob *testNode,
 	fundingOutPoint *wire.OutPoint) {
-	state, _, err := alice.fundingMgr.getChannelOpeningState(fundingOutPoint)
-	if err != ErrChannelNotFound {
-		t.Fatalf("expected to not find channel state, but got: %v", state)
-	}
 
-	state, _, err = bob.fundingMgr.getChannelOpeningState(fundingOutPoint)
-	if err != ErrChannelNotFound {
-		t.Fatalf("expected to not find channel state, but got: %v", state)
+	assertErrChannelNotFound(t, alice, fundingOutPoint)
+	assertErrChannelNotFound(t, bob, fundingOutPoint)
+}
+
+func assertErrChannelNotFound(t *testing.T, node *testNode,
+	fundingOutPoint *wire.OutPoint) {
+
+	var state channelOpeningState
+	var err error
+	for i := 0; i < 10; i++ {
+		state, _, err = node.fundingMgr.getChannelOpeningState(
+			fundingOutPoint)
+		if err != nil && err != ErrChannelNotFound {
```

Comment:
I think this would read a bit better as:

```golang
if err == ErrChannelNotFound {
    // Got expected state, return with success.
    return
} else if err != nil {
    t.Fatalf("unable to get channel state: %v", err)
}
```

Reply:
does this have the same behavior as the current assertion? or are you suggesting to make them a single if-else?

Reply:
I think this comment initially was made in `assertErrChannelNotFound` and magically moved here after I made the change 🚶